servant-foreign/test/Servant/ForeignSpec.hs

```diff
@@ -6,9 +6,11 @@ module Servant.ForeignSpec where
 import Data.Monoid ((<>))
 import Data.Proxy
 import Servant.Foreign
+import Servant.API.Internal.Test.ComprehensiveAPI
```

Comment:
Good! We should perhaps add the Auth combinators to the ComprehensiveAPI as well such that the basic auth bug would have been a compiler error.